compiler/rustc_feature/src/active.rs

```diff
@@ -315,6 +315,8 @@ declare_features! (
     (active, abi_msp430_interrupt, "1.16.0", Some(38487), None),
     /// Allows `extern "ptx-*" fn()`.
     (active, abi_ptx, "1.15.0", Some(38788), None),
+    /// Allows `extern "riscv-interrupt-m" fn()` and `extern "riscv-interrupt-s" fn()`.
+    (incomplete, abi_riscv_interrupt, "CURRENT_RUSTC_VERSION", Some(111889), None),
```

Comment:
What exactly makes the feature `incomplete`? Is there something in particular that doesn't work?

Reply:
Not to my knowledge, I wasn't sure what the right lifecycle for the feature was. I assumed it would start `incomplete` and then be marked `active` at some future point, once *waves hands* criteria were met. 

I read you to be saying that the criterion here is my own confidence in the feature, is that right? If that's the case, it sounds like I should mark it as "active," right?

Reply:
Think of it this way: If this were to go insta-stable, is the feature ready? Does it have any known bugs/limitations, incomplete features, or unsoundness? If not, then it's probably not incomplete.

Reply:
Thanks, that helps: there's no known limitations (outside what's already under thorough discussion on #3246), and it's in a similar boat to those other architectures re: soundness. I note those are all "active," too, so I'll go ahead and flag this one as such also.